app/assets/stylesheets/partials/anime-page.css.scss

```diff
@@ -701,4 +701,3 @@
     line-height: 38px;
   }
 }
-
```

Comment:
I don't know where this change comes from and I couldn't get rid of it.
Somehow this line is there when I few the file on my computer.